code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1888,9 +1888,17 @@ mob/living/carbon/human/isincrit()
 		return FALSE
 	if(isUnconscious() || stunned || paralysis || !check_crawl_ability() || pulledby || locked_to || client.move_delayer.blocked())
 		return FALSE
-	var/crawldelay = round(1 + base_movement_tally()/5) * 1 SECONDS
+	var/crawldelay = 0.2 SECONDS
+	if (crawlcounter >= max_crawls_before_fatigue)
+		if (prob(10))
```

Comment:
This only makes the message have a 10% chance of showing up when you become fatigued while the rest of the proc happens, is this the intended feature?

Reply:
Yes, to avoid spamming your chat if it

Reply:
So the message still happens even when you cannot crawl but still click?

Reply:
If you can't crawl due to fatigue, there's just a 10% chance of the message to appear
I don't see what's so confusing about it.

Reply:
I really don't know why prob(10) is there to avoid chat spam if there already is a check for crawling that returns false if you aren't eligible to crawl. If this is a cosmetic thing, then it's understandable, though I was under the impression there would be some sort of warning when you crawl around too much

Reply:
game so RNG even our error messages are chanced

Reply:
Muh chat spam

Reply:
rng warning messages are hilariously dumb
alternatively you could just like not click
or use goonchat's line combining